oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/measure/BanyanDBMetadataQueryDAO.java

```diff
@@ -271,6 +271,12 @@ public List<Process> listProcesses(String agentId) throws IOException {
                     @Override
                     protected void apply(MeasureQuery query) {
                         query.and(eq(ProcessTraffic.AGENT_ID, agentId));
+                        if (startPingTimeBucket > 0) {
```

Comment:
Could we remove these two IFs to make them mandatory?

Reply:
Fixed.